### DISCUSSION TITLE
`klog stop` should check yesterday’s record as well

### DISCUSSION THREAD

**Discussion by zzehli** [jotaen/klog#380] [Bugs]
If there is no records for the current date, then you would get a error message:
```
Error: No eligible record at date 2021-04-17
Please make sure the record exists and it contains an open-ended time range which start time is prior to your desired end time.
```

You receive a similar error msg when you start tracking the day before and `klog stop` in the next day. Then I have to edit the entry for both dates like the following:

```
2021-03-28
    22:05 - 23:59 Task 1

2021-03-29
    00:00 - 00:57 Task 1 continue
```
It would be great if the new date is entered automatically and as well as a new entry with open ended time range starting at `00:00`

**Comment by jotaen** [comment]
You actually don’t have to split the records in that way, but you can use [the day shifting syntax](https://klog.jotaen.net/#day-shifting) like so:

```
2021-03-28
    22:05 - 0:57>   Task 1
```

The `>` suffix indicates that the time is supposed to refer to the next day. Alternatively you could also do:

```
2021-03-29
    <22:05 - 0:57    Task 1
```

There is an issue in the CLI tool, though, because when you do `klog stop` it doesn’t look for an open range at yesterday’s date, but it only checks today’s date. So if you have:

```
2021-03-28
    22:05 - ?
```

and you do `klog stop` on the 29th, it complains. That should be fixed.

**Comment by jotaen** [comment]
Tracked the other issue separately, thanks for reporting. https://github.com/jotaen/klog/issues/60

**Comment by zzehli** [comment]
thanks!

**Comment by jotaen** [comment]
I leave this ticket here open as well, to track the issue with `klog stop` taking yesterdays date into consideration.

**Comment by jotaen** [comment]
Fixed through https://github.com/jotaen/klog/pull/76
